chapters/ch07.md

```diff
@@ -1,27 +1,27 @@
 
 ->
-Trong các ứng dụng quan trọng và đã khá phát triển - lấy ví dụ như quảng cáo, tìm kiếm trên web và đề xuất sản phẩm - tôi đã thấy nhiều nhóm có động lực cao để cải thiện thậm chí chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi nhuận của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000, để phát hiện được những cải thiện còn nhỏ hơn nữa.
+Trong các ứng dụng quan trọng và đã khá phát triển - lấy ví dụ như quảng cáo, tìm kiếm trên web và gợi ý sản phẩm - tôi đã thấy nhiều nhóm rất muốn cải thiện thậm chí chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi nhuận của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000 để phát hiện được những cải thiện còn nhỏ hơn nữa.
 
 How about the size of the test set? It should be large enough to give high confidence in the overall performance of your system. One popular heuristic had been to use 30% of your data for your test set. This works well when you have a modest number of examples—say 100 to 10,000 examples. But in the era of big data where we now have machine learning problems with sometimes more than a billion examples, the fraction of data allocated to dev/test sets has been shrinking, even as the absolute number of examples in the dev/test sets has been growing. There is no need to have excessively large dev/test sets beyond what is needed to evaluate the performance of your algorithms.
 
 ->
-Vậy còn kích thước của tập đánh giá thì sao? Nó phải đủ lớn để mang lại sự tự tin cao về hiệu năng tổng thể trong hệ thống của bạn. Một quy tắc chung khá phổ biến đó là sử dụng 30% dữ liệu của bạn cho tập đánh giá. Như vậy thì ổn khi bạn có số lượng data khiêm tốn, từ 100 đến 10.000 chẳng hạn. Nhưng trong thời đại Big data hiện nay mà những bài toán machine learning đôi khi có hơn một tỷ ví dụ, phần dữ liệu được phân bổ cho các tập đánh giá / phát triển đã bị thu hẹp lại, mặc dù số lượng tuyệt đối các ví dụ trong các tập phát triển / đánh giá vẫn đang tăng lên. Bạn không cần phải có tập phát triển / đánh giá lớn hơn mức cần thiết để đánh giá hiệu năng các thuật toán của bạn.
+Vậy còn kích thước của tập kiểm tra thì sao? Nó phải đủ lớn để mang lại độ tin cậy cao về hiệu năng tổng thể trong hệ thống của bạn. Một quy tắc từ thực nghiệm khá phổ biến đó là sử dụng 30% dữ liệu của bạn cho tập kiểm tra. Như vậy thì ổn khi bạn có số lượng data khiêm tốn, từ 100 đến 10.000 chẳng hạn. Nhưng trong thời đại Big data hiện nay mà những bài toán machine learning đôi khi có hơn một tỷ mẫu, phần dữ liệu được phân bổ cho các tập phát triển/kiểm tra đã bị thu hẹp lại, dù cho số lượng tuyệt đối mẫu trong tập phát triển/kiểm tra vẫn đang tăng lên. Bạn không cần tập phát triển/kiểm tra lớn hơn mức cần thiết để đánh giá hiệu năng thuật toán.
```

Comment:
Mình đề xuất sửa như sau 
- **Should be** nên dịch là "cần" thì sát nghĩa hơn là phải. Vì một số trường hợp là không cần thiết. 
- Mình có băn khoăn về từ "performance" này không nên dịch là hiệu năng, mà nên dịch là chất lượng thì đúng hơn. @tiepvupsu anh @sonvx vào review xem nhé
-  "tổng thể ~~trong~~ của hệ thống ~~của bạn~~." 
- đề xuất 
> Một quy tắc từ thực nghiệm khá phổ biến đó là sử dụng 30% dữ liệu của bạn cho tập kiểm tra. Như vậy thì ổn khi bạn có số lượng data khiêm tốn, từ 100 đến 10.000 chẳng hạn.

Thành
> Với tập dữ liệu khiêm tốn từ 100 tới 10.000 mẫu, thì dùng 30% dữ liệu làm tập kiểm tra là một quy tắc thực nghiệm khá phổ biến.  
* "thời đại" --> kỷ nguyên. 

> Nhưng trong ~~thời đại~~ kỷ nguyên của big data ~~hiện nay~~, **khi có** những bài toán machine learning có tới hơn một tỷ mẫu, ~~phần dữ liệu được phân bổ cho các~~ **tỷ lệ (phân chia)** tập phát triển/kiểm tra đã bị thu hẹp lại **mặc** dù ~~cho~~ số lượng ~~tuyệt đối~~ mẫu trong tập phát triển/kiểm tra vẫn ~~đang~~ tăng lên.

> Thực sự không cần tỷ lệ tập phát triển/kiểm tra lớn hơn mức cần thiết - mức đánh giá ~~hiệu năng~~ chất lượng thuật toán.